### DISCUSSION TITLE
Turing Pi anyone?

### DISCUSSION THREAD

**Discussion by brendanobra** [blakeblackshear/frigate#5482] [General]
I run Frigate (and other stuff) on Kubernetes (k0s) and want to lower my heat/power footprint - it's currently running a full size 8 core/32GB tower with 400w power supply. Nice in the winter, but not so nice in the summer.
I am running one Coral USB, and it works great with this setup.

I'd like to transition to Turing PI running rPI CM4s (using the Coral USB).  Does anyone have any experience with this setup (or anything else that i should be aware of).
TIA

**Comment by Cold-Lemonade** [comment]
@brendanobra Were you able to successfully install Frigate on your Turing Pi? How is it working for you?

**Comment by brendanobra** [comment]
Sadly, I do not have a TuringPi yet. The availability of CM4s (and the
effect on pricing) puts it out of reach for at least a few months for me.
Looking forward to it for sure tho,but I'm also wondering if I will need to
add a second Coral USB. You?

On Fri, Jun 2, 2023 at 4:20 PM andrew-lemon ***@***.***>
wrote:

> @brendanobra <https://github.com/brendanobra> Were you able to
> successfully install Frigate on your Turing Pi? How is it working for you?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/blakeblackshear/frigate/discussions/5482#discussioncomment-6076263>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFUZX3NKIOZX2HDXXQENLDXJJYM3ANCNFSM6AAAAAAU2WFMSU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>


-- 
GVoice: 707.410.0371
LinkedIn: http://www.linkedin.com/in/brendanobra

**Comment by Cold-Lemonade** [comment]
I don't have a TuringPi either. That's why I was curious about your experience with it.

I'm running Frigate on a Minisforum GK41, which has a J4125 processor. I have two Coral USBs, but I am not exactly sure how beneficial having the second one is. I haven't noticed any appreciable change in performance with and without the second one.

**Reply by NickM-27** [reply]
There will not be any performance benefit to multiple corals unless your first corals are fully saturated and you have skipped fps

**Comment by brendanobra** [comment]
Thanks!

On Sat, Jun 3, 2023 at 7:14 AM Nicolas Mowen ***@***.***>
wrote:

> There will not be any performance benefit to multiple corals unless your
> first corals are fully saturated and you have skipped fps
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/blakeblackshear/frigate/discussions/5482#discussioncomment-6078949>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFUZX2HSLRQOGBKKN3XUP3XJNBFTANCNFSM6AAAAAAU2WFMSU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>


-- 
GVoice: 707.410.0371
LinkedIn: http://www.linkedin.com/in/brendanobra

**Comment by hagak** [comment]
Well I have a turing pi v2 but sadly no CM4 modules.  I jumped on the kickstarter thinking that by the time the boards ship surely i could get hold of a few CM4 modules.  NOPE!

Primary goal is to run K3s cluster on it, however not sure if I will move frigate to it or not.  Most likely will at least test and compare it to my current setup though.  But who the heck knows when I can get any CM4 modules.

**Reply by brendanobra** [reply]
Yeah, I'm in the same boat, hoping to stand one up by 2024 at this rate.

**Reply by didn0t** [reply]
Have a look on the TPi Discord Buy/Sell thread - there are a few people selling them at reasonable prices - I've selling mine.
